doc/en/how-to/output.rst

```diff
@@ -270,6 +270,10 @@ situations, for example you are shown even fixtures that start with ``_`` if you
 Using higher verbosity levels (``-vvv``, ``-vvvv``, ...) is supported, but has no effect in pytest itself at the moment,
 however some plugins might make use of higher verbosity.
 
+By default for verbosity inferior to ``-vv`` really long output are truncated. It's also possible to
+control the truncation directly using the ``assert_truncate_level`` option. You can use ``assert_truncate_level=0``
```

Comment:
By using the `confval` as I mentioned before, we can create a link to it automatically:

```suggestion
control the truncation directly using the :confval:`assert_truncate_level` option. You can use ``assert_truncate_level=0``
```